ccd-definition/CaseField/CareSupervision/DEPRECATED.json

```diff
@@ -372,6 +372,16 @@
     "SecurityClassification": "Public",
     "Searchable": "N"
   },
+  {
```

Comment:
Might want to double check if we want to do this now

Reply:
This should be fine, I don't think it does anything special too much - the field will still 'exist' for CCD purposes